tests/classes/export/grading_sheet_download_test.php

```diff
@@ -72,48 +72,48 @@ public function test_one_stage_no_allocations() {
         $submission = $this->submission;
 
         // headers and data for csv
-        $csv_cells = array('submissionid', 'submissionfileid', 'name', 'username', 'submissiontime', 'singlegrade', 'feedbackcomments');
+        $csvcells = ['submissionid', 'submissionfileid', 'name', 'username', 'submissiontime', 'singlegrade', 'feedbackcomments'];
 
         $timestamp = date('d_m_y @ H-i');
         $filename = get_string('gradingsheetfor', 'coursework'). $this->coursework->name .' '.$timestamp;
-        $grading_sheet = new \mod_coursework\export\grading_sheet($this->coursework, $csv_cells, $filename);
-        $actual_submission = $grading_sheet->add_cells_to_array($submission, $student, $csv_cells);
+        $gradingsheet = new \mod_coursework\export\grading_sheet($this->coursework, $csvcells, $filename);
```

Comment:
same question about variable & namespace renaming

Reply:
This looks ok